docs/src/doc/user-guide/advanced/graal-vm-native-image.md

```diff
@@ -17,4 +21,24 @@ godot {
 ```
 
 In order to use the generated native-image, you can pass `--java-vm-type=graal` argument to engine, or simply change
-`godot_kotlin_configuration.json` to set vm_type to `graal`.
\ No newline at end of file
+`godot_kotlin_configuration.json` to set vm_type to `graal`.
+
+## Reflexion and JNI with native image
```

Comment:
Typo for whole paragraph: Reflection
Also i suggest the title: `Reflection, libraries and JNI with native image`